core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -3825,6 +3802,36 @@ protected void addInitialAdminData(boolean addAdmin, boolean initialSetup) throw
         doInitialUserAdding();
     }
 
+    private void validateUsernamePassword(String adminUserName, Object adminPassword) throws UserStoreException {
```

Comment:
Re-factor adminUserName, adminPassword variables to userName, credential.

Because, this method is used by addUser() method and in that flow it will be used to create non-admin users as well.